ngraph/frontend/tensorflow/src/ngraph_conversions.hpp

```diff
@@ -78,16 +76,16 @@ void NCHWtoHW(const std::vector<T>& src, std::vector<size_t>& dst) {
 }
 }  // namespace detail
 
-void NHWCtoNCHW(const std::string& op_name, bool need_convert, ov::Output<ov::Node>& ng_input);
+void convert_nhwc_to_nchw(const std::string& op_name, bool need_convert, ov::Output<ov::Node>& ng_input);
 
-void NCHWtoNHWC(const std::string& op_name, bool need_convert, ov::Output<ov::Node>& ng_node);
+void convert_nchw_to_nhwc(const std::string& op_name, bool need_convert, ov::Output<ov::Node>& ng_node);
 
 template <typename T>
-void NHWCtoHW(bool is_nhwc, const std::vector<T>& src, std::vector<size_t>& dst) {
+void convert_nhwc_to_hw(bool is_nhwc, const std::vector<T>& src, std::vector<size_t>& dst) {
     if (is_nhwc) {
-        detail::NHWCtoHW(src, dst);
+        detail::convert_nhwc_to_hw(src, dst);
     } else {
-        detail::NCHWtoHW(src, dst);
+        detail::convert_nchw_to_hw(src, dst);
```

Comment:
the same comment. Propose to remove useless routines. And rename to avoid confusion

Reply:
need to be analyzed in more detail
let's do it in the next PR separately